src/mappings/content/video.ts

```diff
@@ -169,6 +186,28 @@ export async function processVideoDeletedByModeratorEvent({
   await deleteVideo(overlay, contentId)
 }
 
+export async function processVideoAssetsDeletedByModeratorEvent({
+  block,
+  indexInBlock,
+  extrinsicHash,
+  overlay,
+  event: {
+    asV1000: [deletedBy, contentId, assetIds, rationale],
+  },
+}: EventHandlerContext<'Content.VideoAssetsDeletedByModerator'>): Promise<void> {
+  const video = await overlay.getRepository(Video).getByIdOrFail(contentId.toString())
+
+  overlay.getRepository(Event).new({
+    ...genericEventFields(overlay, block, indexInBlock, extrinsicHash),
+    data: new VideoAssetsDeletedByModeratorEventData({
+      video: video.id,
+      assetIds,
+      deletedBy: parseContentActor(deletedBy),
+      rationale: rationale.toString(),
+    }),
+  })
```

Comment:
Could you please check whether is it suitable to also remove the related `StorageDataObject` rows starting from `assetIds`. If you go through the `processVideoDeletion` I believe data objects are not deleted but in the rutnime they are.
So maybe the same considerations apply also there